src/rime/dict/vocabulary.h

```diff
@@ -30,20 +30,36 @@ class Code : public vector<SyllableId> {
   string ToString() const;
 };
 
+struct ShortDictEntry {
```

Comment:
How much more memory efficient is this?

If the diff by this single change is insignificant, I'd prefer not to repeat the code.

Reply:
The screenshots I pasted in the Additional Info have showed the info you get interested. In my test case, it may save about 15 % ~ 18 % of the memory consumption.